src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/ChimericAlignment.java

```diff
@@ -204,6 +229,23 @@ public static StrandSwitch determineStrandSwitch(final AlignmentInterval first,
     }
 
     /**
+     * An SV event could be detected from a contig that seem originate from the forward or reverse strand of the reference,
+     * besides the annotation that the alignment flanking regions might flank either side of the two breakpoints.
+     */
+    @VisibleForTesting
+    static boolean isForwardStrandRepresentation(final AlignmentInterval regionWithLowerCoordOnContig,
```

Comment:
not quite understanding the commit. Could you explain a bit more?

Reply:
I was referring to what I said above:

>Also, the fact that there's a method called isForwardStrandRepresentation that's used in the other half of this if/else but doesn't handle the logic for this half bugs me a bit. Wouldn't it be possible to extend that method to handle this logic? Alternatively I'd rename that method to something that describes the scenario in which it's applicable, eg `isIntraContigMappingForwardStrandRepresentation`.

To clarify what I meant, the entire chunk of code in ChimericAlignment from lines 119 to 130 seems to be determining whether the alignment is in "forward strand representation." Part of that chunk of code delegates to this method called `isForwardStrandRepresentation`. I feel like the naming is confusing because this method is only handling the logic for a specific subset of inputs, but it would be a natural place to go for a code reader who wanted to see where "forward strand representation" is being set.

Reply:
Ah, I see. Thanks! I just responded to the comment above with some docs in the method, and some figures.
The incoherence has been handled in a separate branch that also tries to deal with several other issues with this class. It will generate another small PR with tests.